File: Daily Challanges/2026-02-03.py
def mirror(s):
    return s + s[::-1]

test1 = mirror("freeCodeCamp")
test2 = mirror("RaceCar")
test3 = mirror("helloworld")
test4 = mirror("The quick brown fox...")

check1 = "freeCodeCamppmaCedoCeerf"
check2 = "RaceCarraCecaR"
check3 = "helloworlddlrowolleh"
check4 = "The quick brown fox......xof nworb kciuq ehT"

print(f"Test1: {test1 == check1}")
print(f"Test2: {test2 == check2}")
print(f"Test3: {test3 == check3}")
print(f"Test4: {test4 == check4}")

File: Daily Challanges/2026-02-February/2026-02-03.js
function mirror(str) {
  return str + str.split('').reverse().join('');
}

const test1 = mirror("freeCodeCamp");
const test2 = mirror("RaceCar");
const test3 = mirror("helloworld");
const test4 = mirror("The quick brown fox...");

const check1 = "freeCodeCamppmaCedoCeerf";
const check2 = "RaceCarraCecaR";
const check3 = "helloworlddlrowolleh";
const check4 = "The quick brown fox......xof nworb kciuq ehT";

console.log(`Test1: ${test1 === check1}`);
console.log(`Test2: ${test2 === check2}`);
console.log(`Test3: ${test3 === check3}`);
console.log(`Test4: ${test4 === check4}`);